python.md

```diff
@@ -566,10 +567,31 @@ class CreateSomething(generic.FormView):
                 bar=form.cleaned_data['bar'],
             )
         except domain.UnableToCreateThing as e:
+            # Handle *anticipated* exceptions; things we know might go wrong but
+            # can't do much about (eg vendor errors). Here we assume the exception 
+            # message is suitable for end-users but often it needs some
+            # adjusting.
             form.add_error(None, str(e))
             return self.form_invalid(form)
-
-        # Handle successful creation...
+        except Exception as e
+            # Optional handling of *unanticipated* errors. When this happens, we want to send
+            # details of the error to Sentry so such errors can be investigated
+            # and fixed. It's optional as we could omit this block and let the
+            # exception percolate up to generate a 500 response which Sentry
+            # will capture automatically. It's a bit friendlier to the user to
+            # show them some custom copy rather than a generic 500 response.
+            #
+            # Depending on context, it might not be appropriate to
+            # show the exception message to the end-user. If possible, offer
+            # some guidance on what the end-user should do next (eg contact
+            # support, try again, etc)
+            msg = "Really sorry, we weren't able to do the thing as an unanticipated error occurred. Error message: %s" % e
```

Comment:
This line is a bit long for reading on the markdown page in Github.

Reply:
Yep - adjusted